### ISSUE TITLE
Kics reports an error on Ansible commented values

### ISSUE THREAD

**Issue by LvffY** [Checkmarx/kics#4419]
### Expected Behavior

**Kics** should not report that a he finds a password in my file.

### Actual Behavior

Beside my value is just an example and is commented, **kics** reports an error

### Steps to Reproduce the Problem

1.  Have the following **group_vars/backends/vars.yml**

```yaml
---
backends:
  db_name: "db"
  user:
    username: "user"
#    definition of using host_vars variable in an other variable
#    hostvars:
#      airflow_1:
#        vault_demo_airflow_user_1_password: value
    password: "{{ another_encrypted_value }}"

```

2.  In `checkmarx/kics:debian` run the analysis `/app/bin/kics scan -p group_vars/backends/vars.yml -o ${PWD} --report-formats json,sarif --ignore-on-exit results --verbose --log-level=DEBUG`

3.  [Obtain this kind of output ](https://github.com/Checkmarx/kics/files/7394785/analysis.txt)

### Specifications

-   Version: 1.4.5
-   Platform: Ansible
-   Subsystem: N/A

**Comment by rafaela-soares** [comment]
Hello, @LvffY 🙂 

I hope this message finds you well. Thank you so much for reaching us.

We are happy to inform you that we all solving it!
